src/components/picker/picker.tsx

```diff
@@ -321,14 +321,14 @@ export class Picker {
     }
 
     private createChip(listItem: ListItem): Chip {
+        const name = getIconName(listItem.icon);
+        const color = getIconFillColor(listItem.icon, listItem.iconColor);
```

Comment:
Could we use `listItem.icon?.color` here instead? Since `iconColor` is marked as deprecated

Reply:
we should have both since `iconColor` still in used